js/src/sheet.ts

```diff
@@ -308,7 +308,7 @@ let SheetView = widgets.DOMWidgetView.extend({
         for (let key in this.widget_views) {
             if(this.widget_views.hasOwnProperty(key)) {
                 // Ugly, this should be properly done
-                let [row, col] = String(key).split(',').map(parseInt);
+                let [row, col] = String(key).split(',').map(x => parseInt(x));
```

Comment:
Is the problem that map calls parseInt with a lot of arguments, but really we just want parseInt to be called with the first argument, just the value?

Reply:
Oh wow, parseInt was interpreting the value as having radix whatever the index in the list was? Tricky!

Reply:
I don't understand what is going on on this. I only found the issue but don't understand it:
![map_parseint](https://user-images.githubusercontent.com/21197331/53753115-c31af200-3eb0-11e9-9671-1300fc04607f.png)

I don't think it has something to do with radix, does it? I don't really understand what you understood :P

Reply:
Ok, reading the documentation of `Array.map` I understand now :) That was tricky indeed :P